### PR TITLE
Adds support for Debian Buster

### PR DESCRIPTION
**Pull request by Sprinterfreak**
* Add debian-buster to supported os list
* Add php-fpm7.3 environment meta
* Add nginx template client_body_timeout avoids web-updater getting killed by nginx

### CODE REVIEW

**Comment by conorsch** [comment]
@Sprinterfreak Thank you for submitting this! It wasn't quite what I needed in order to get Buster support on my setup. Can you take a look at https://github.com/aalaesar/install_nextcloud/pull/81 and see if that meets your needs, as well?

**Comment by staticdev** [comment]
@aalaesar this should be closed. #81 already adds it.

File: meta/main.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ galaxy_info:
   - name: Debian
     versions:
     - stretch
+    - buster
```
Comment (staticdev) [inline meta/main.yml:19]: This is already supported.

**Review by staticdev — CHANGES_REQUESTED** [review]
@Sprinterfreak please close this PR, it was fixed on #81.

**Pull request closed without merging**